tests/headers.rs

```diff
@@ -67,7 +67,8 @@ fn check_file_deps<P: AsRef<Path>>(
         if let Some(entries) = files_deps.get_mut(&filename) {
             let file_content = read_file(r_p);
             for line in file_content.lines() {
-                if !line.starts_with("use ") && !line.starts_with("pub use ") {
+                if (!line.starts_with("use ") && !line.starts_with("pub use "))
+                    || line.starts_with("use core::") {
```

Comment:
Instead of modifying the test, just `use core::whatever;` inside the function that needs it.